experimental/cotel/otel.go

```diff
@@ -0,0 +1,29 @@
+package cotel
```

Comment:
is the idea here to eventually move all otel apis into this package?  I"d be fine with that.  Either way, I think a readme describing the purpose of this package, and its expectations for getting out of experimental status, would help a lot.

Reply:
Could I ask for a readme in /experimental with a short statement like, "subpackages in this dir are undergoing evaluation and may be migrated to a non-experimental package at a later date.  Movement out of /experimental will always be a two step deprecation process to allow users to adapt."